libsrc/osic1p/cputc.s

```diff
@@ -36,10 +41,13 @@ L1:     cmp     #$0D            ; LF?
 cputdirect:
```

Comment:
Ok, at the moment 100% of the known users of the osic1p target are willing to accept slightly larger cputdirect code for getting support for scrolling :grimacing:. Let's talk about removing scrolling support from conio once we have POSIX I/O.